configure.ac

```diff
@@ -16,9 +17,12 @@ AM_PROG_CC_C_O
 AC_DISABLE_STATIC
 AC_PROG_LIBTOOL
 
-dnl i18n
-IT_PROG_INTLTOOL([0.35.0])
-GP_I18N
+dnl i18n, inline so autoreconf can detect gettext usage
+#AM_GNU_GETTEXT_VERSION([0.19.8])
+AM_GNU_GETTEXT_REQUIRE_VERSION([0.19.8])
```

Comment:
what's up with this in the end?

Reply:
See my comment in https://github.com/geany/geany-plugins/pull/1434#discussion_r2105801339

Reply:
Oops, my bad, I got lost with all these.
OK, looking at [the docs](https://www.gnu.org/software/gettext/manual/html_node/autopoint-Invocation.html) it seems like using the latter is fine: AFAIK we don't really care which version is used so long as it works as expected.
Also, this macro seems [old enough](https://lists.gnu.org/archive/html/autoconf-patches/2015-10/msg00000.html) for our requirements I guess

Reply:
So I'd say just drop the commented-out `AM_GNU_GETTEXT_VERSION` and call it good

Reply:
Alright, done.